tracer.go

```diff
@@ -205,6 +213,10 @@ func (t *Tracer) startSpanWithOptions(
 		if hasParent && parent.isDebugIDContainerOnly() {
 			ctx.flags |= (flagSampled | flagDebug)
 			samplerTags = []Tag{{key: JaegerDebugHeader, value: parent.debugID}}
+			textMapPropagator, ok := t.injectors[opentracing.TextMap].(*textMapPropagator)
+			if ok {
+				samplerTags = []Tag{{key: textMapPropagator.headerKeys.JaegerDebugHeader, value: parent.debugID}}
```

Comment:
no, that one is needed, but the lines you added are not - the tag should always be `jaeger-debug`, it does not need to change because http headers were reconfigured